bin/faucet/Cargo.toml

```diff
@@ -29,6 +29,7 @@ miden-tx           = { workspace = true, features = ["concurrent"] }
 rand               = { workspace = true }
 rand_chacha        = "0.9"
 serde              = { version = "1.0", features = ["derive"] }
+sha3               = { version = "0.10.8" }
```

Comment:
Question: what's the motivation for using SHA3? For non-arithmetizaton-friendly hashing we usually use BLAKE3 (which is also available through `miden-crypto`).

Reply:
I defaulted to SHA3 since it is pretty standard and found its slowness helpful in this this case (also found a standardize JS version was really easy).
I dig a bit more after your comment and saw that using blake3 in the JS is possible two. If we have some preference with blake, I can replace it 👍🏼

Reply:
I'm trying to replace it with blake, getting a functional version isn't resulting as easy as i thought i was going to be. Rust side is done, but in the JS side is a bit harder that I initially thought (though I'm not really a JS savvy, will ask someone with more knowledge)